gsdk/Cargo.toml

```diff
@@ -45,6 +45,7 @@ gsdk = { path = ".", features = ["testing"] }
 tokio = { workspace = true, features = [ "full" ] }
 demo-messenger.workspace = true
 demo-new-meta.workspace = true
+demo-ping = { workspace = true, features = ["std"] }
```

Comment:
Probably I would rather use one of the already referenced demos in order to avoid compilation time increase. The `demo-new-meta` seems having some suitable message handling.

Reply:
I looked at `demo-messenger`, but it doesn't contain methods with `msg::reply_bytes`, so I took `ping`

Reply:
but I'm gonna try it with `demo-new-meta`